src/hooks/useTxDetails/useTxDetails.ts

```diff
@@ -91,7 +91,7 @@ export const useTxDetails = (txId: string, activeAsset?: Asset): TxDetails => {
   const buyTransfer = getTransferByType(tx, chainAdapters.TxType.Receive)
   const sellTransfer = getTransferByType(tx, chainAdapters.TxType.Send)
   const tradeTx = (activeAsset && getTransferByAsset(tx, activeAsset)) ?? buyTransfer
-  // const tradeTx = activeAsset?.caip19 === sellTransfer?.caip19 ? sellTransfer : buyTransfer
+  // const tradeTx = activeAsset?.assetId === sellTransfer?.caip19 ? sellTransfer : buyTransfer
```

Comment:
if we're changing a comment we should just remove it

Reply:
I don't know why this line was commented instead of removed in the PR that did it - let's delete it.